server/infrastructure/portal/client/PortalClient.ts

```
@@ -31,9 +31,16 @@
       body: JSON.stringify({ username, password }),
     });
 
+
     if (!response.ok) {
-      const error = await response.json();
-      throw new Error(error.message || '크롤링 중 오류가 발생했습니다.');
+      const result = await response.json();
```

ESLint

[error] 'result' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)